packages/compass-sidebar/src/components/multiple-connections/saved-connections/saved-connection.tsx

```diff
@@ -57,15 +59,33 @@ type Action =
   | 'duplicate-connection'
   | 'remove-connection';
 
+const WarningIcon = () => {
```

Comment:
There is a leafygreen warning icon, why are we making a copy?

Reply:
Didn't realise. Fixing it.